docs/reference-guides/best-practices/rancher-server/tips-for-scaling-rancher.md

```diff
@@ -0,0 +1,50 @@
+
+Without using ACE, all kubeconfig requests first route through Rancher. This contributes to load on the Rancher setup, while using ACE completely bypasses the Rancher setup.
+
+### Option to Reduce Event Handler Executions
+The bulk of Rancher's logic are ran by event handlers. These event handlers run on an object whenever the object is updated. They also run when Rancher is started. By default they run every 15 hours when caches are synced but this can be changed using the CATTLE_SYNC_ONLY_CHANGED_OBJECTS environment variable. If resource allocation spikes are seen on an interval of about 15 hours it is possible this setting can help. The value for the environment variable can be a comma separated list of the following options: mgmt, user, and scaled. The values refer to types of controllers (the structures that contain and run handlers) and their handlers. Mgmt refers to management which only runs on one Rancher node. User will run for every cluster. Some user handlers run on a Rancher node while others run downstream in their respective cluster. Scaled runs on every Rancher node. The two options that cover most controllers' handlers and are supported are mgmt and user. There are very few "scaled" controllers and their handlers are often critical. It is not recommended that the scaled value is used.
+
+## Optimizations Outside of Rancher
+A large component of performance is the local cluster and how it was configured. Since Rancher is "Bring your own cluster", this impactful component introduces a potential bottleneck before Rancher software ever runs. When Rancher nodes experience high resource usage, you can use the command "top" to identify whether it is Rancher or a kubernetes component that is consuming the resource in excess.
+
+### Keeping Kubernetes Versions Up to Date
+Similar to Rancher versions, it is advisable to keep your kubernetes cluster up to date. This will ensure that your cluster contains any available performance enhancements or bug fixes.
+
+### Optimizing ETCD
+As mentioned [here](https://etcd.io/docs/v3.4/op-guide/performance/), the two main bottlenecks to ETCD performance are disk speed and network speed. Optimization to either should improve performance. For information regarding ETCD performance see [Slow etcd performance (performance testing and optimization)](https://www.suse.com/support/kb/doc/?id=000020100). Information on disk can also be found [here](https://docs.Ranchermanager.Rancher.io/v2.5/pages-for-subheaders/installation-requirements#disks). Theoretically, the more nodes in an ETCD cluster the slower it will be due to replication requirements [source](https://etcd.io/docs/v3.3/faq). It can be inferred that ETCD performance will be inversely affected by distance between nodes as that will slow down network communication.
```

Comment:
* Good call including that KB link.
* Break up bottom paragraph
* Using "here" as a link is considered bad UX, "source" is fine though

```suggestion
The two main bottlenecks to [ETCD performance](https://etcd.io/docs/v3.4/op-guide/performance/) are disk speed and network speed. Optimization to either should improve performance. For information regarding ETCD performance see [Slow etcd performance (performance testing and optimization)](https://www.suse.com/support/kb/doc/?id=000020100). Information on disks can also be found [in our docs](https://docs.Ranchermanager.Rancher.io/v2.5/pages-for-subheaders/installation-requirements#disks). 

Theoretically, the more nodes in an ETCD cluster the slower it will be due to replication requirements [source](https://etcd.io/docs/v3.3/faq). This may be counter-intuitive to common scaling approaches. It can also be inferred that ETCD performance will be inversely affected by distance between nodes as that will slow down network communication.
```